App/routes/components/ProjectMenu.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react'
+import { View, Image, StyleSheet } from 'react-native'
+import {
+  Menu,
+  MenuOptions,
+  MenuOption,
+  MenuTrigger,
+} from 'react-native-popup-menu';
+import { useDispatch, useSelector } from 'react-redux';
+import { getCurrentProjectUuid, getCurrentProject, renameProject } from '../../state/projects';
+import RenameProjectModal from './RenameProjectModal'
+
+
+export default () => {
+  const dispatch = useDispatch()
+  const [modalVisible, setModalVisible] = useState(false)
+  const projectUuid = useSelector(getCurrentProjectUuid)
+  const project = useSelector(getCurrentProject)
+  const renameProjectHandler = (newName) => {
+    dispatch(renameProject({ projectUuid, newName })) // should be called when hit save on opened modal
+  }
+  const options = [
+    {
+      label: 'Rename',
+      callback: () => setModalVisible(true)
+    }
+  ]
+  const menuOptions = options.map(option => <MenuOption onSelect={option.callback} text={option.label} />)
+  return <View style={styles.wrapper}>
+    <Menu>
+      <MenuTrigger customStyles={{ border: 0, borderRadius: 8 }}>
+        <Image source={require('../../assets/hamburger-icon.png')} style={{ width: 20, height: 20 }} />
```

Comment:
import image on top lines and pass here as variable

import hamburgerIcon from './icon/path/
....
<Image source={hamburgerIcon} />